chsdi/models/vector/bafu.py

```diff
@@ -11,6 +11,19 @@
 Base = bases['bafu']
 
 
+class hintergrundkarte(Base, Vector):
```

Comment:
That's not consistent at all... should we go through every model and change to uppercase every class?